src/commands/game.js

```diff
@@ -9,15 +9,13 @@ const createRockPaperScissors = require('../creators/rock-paper-scissors');
 const game = (ticketId) => {
   getTicketData(ticketId)
     .then(({ data }) => {
-      const { template } = data;
```

Comment:
This doesn't work?

Reply:
My bad, ignore the changes to this file please.
The changes to template.js however, I do stand by.